{
	"class": "vulnerability",
	"id": "oval:com.ubuntu.noble:def:24040047900000",
	"version": "1",
	"metadata": {
		"Title": "glusterfs",
		"reference": {
			"source": "Package",
			"ref_id": "glusterfs",
			"ref_url": "https://launchpad.net/ubuntu/+source/glusterfs"
		},
		"Description": "clustered file-system",
		"affected": {
			"family": "unix",
			"Platform": "Ubuntu 24.04 LTS"
		},
		"advisory": {
			"Rights": "Copyright (C) 2024 Canonical Ltd.",
			"Component": "universe",
			"CurrentVersion": "11.1-4ubuntu0.1",
			"cve": [
				{
					"text": "CVE-2023-26253",
					"href": "https://ubuntu.com/security/CVE-2023-26253",
					"priority": "medium",
					"public": "20230221",
					"cvss_score": "7.5",
					"cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
					"cvss_severity": "high",
					"test_ref": "oval:com.ubuntu.noble:tst:24040047900000",
					"usns": "5989-1,6157-1"
				}
			]
		}
	},
	"criteria": {
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:com.ubuntu.noble:tst:24040047900000",
						"comment": "(CVE-2023-26253) glusterfs package in noble, is related to the CVE in some way and has been fixed (note: '10.3-5')."
					}
				]
			}
		]
	}
}
